lib/rucio/daemons/transmogrifier/transmogrifier.py

```diff
@@ -441,27 +446,27 @@ def transmogrifier(bulk=5, once=False, sleep_time=60):
             for identifier in chunks(identifiers, 100):
                 _retrial(set_new_dids, identifier, None)
 
-            logging.info(prepend_str + 'Time to set the new flag : %f' % (time.time() - time1))
+            logger(logging.DEBUG, 'Time to set the new flag : %f' % (time.time() - time1))
             tottime = time.time() - start_time
             for sub in subscriptions:
                 update_subscription(name=sub['name'], account=sub['account'], metadata={'last_processed': datetime.now()})
-            logging.info(prepend_str + 'It took %f seconds to process %i DIDs' % (tottime, len(dids)))
-            logging.debug(prepend_str + 'DIDs processed : %s' % (str(dids)))
+            logger(logging.INFO, 'It took %f seconds to process %i DIDs' % (tottime, len(dids)))
+            logger(logging.DEBUG, 'DIDs processed : %s' % (str(dids)))
             monitor.record_counter(counters='transmogrifier.job.done', delta=1)
             monitor.record_timer(stat='transmogrifier.job.duration', time=1000 * tottime)
         except Exception:
             exc_type, exc_value, exc_traceback = exc_info()
-            logging.critical(prepend_str + ''.join(format_exception(exc_type, exc_value, exc_traceback)).strip())
+            logger(logging.CRITICAL, ''.join(format_exception(exc_type, exc_value, exc_traceback)).strip())
```

Comment:
here, you could actually just use `exc_info=True` as done here https://github.com/rucio/rucio/blob/f2547ced433d31881ace649cdde72b8c5678208f/lib/rucio/daemons/conveyor/poller.py#L277